template/10-timeline.md

```diff
@@ -1,10 +1,92 @@
 # Timeline/Resource Planning
 
-*What’s the overall schedule you’re working towards?*
+## Execution Roadmap
 
-*What resources are required?*
+Planning an efficient schedule will prevent us from making time-wasting mistakes and reduce costs. The timeline is organized over a 14 week period, marked with 4 milestones. Main guidelines will ensure that the whole development process complies with the MVP requirements. A team of 6 people will be in charge of delivering a product ready for initial rollout: 2 backend developers, 2 frontend developers, 1 UI/UX designer and 1 project manager. The major points dealt with during this development process are setup, development, testing and rollout.
 
-*What are the intermediate milestones?*
+#### Milestone 1
 
-*List identified sprints*
+Initial Setup and First Design Iteration
 
+- Setup of the backend (Google Firebase)
+- Documentation of backend API
+- Basic frontend building blocks
+- First interation of UI/UX design (Figma) 
+- Google login integration
+- Tests with mock datas 
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 1 | Preparation of the codebase| Documentation of the backend API. Global view of the project's visual/technical directions. |
+| Week 2 | First UI/UX Design and Technical Setup | First UI/UX designs, Google Authentification and Firebase setup, basic frontend building blocks |
+
+#### Milestone 2
+
+Post features development
+
+- Develop the core post system that the users will mainly interact with.
+- Ensure that each new feature is thouroghly tested.
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 3 | Post creation | Posts with data can be stored and retrieved from the backend. Frontend elements can display this data. |
+| Week 4 | Commenting system | Completed commenting feature in frontend and backend |
+| Week 5 | Voting system | Developped voting system. Unit testing that ensures that database stays consistent |
+| Week 6 | Location-based feed | Combinations of the developped features in the frontend. Mock testing. |
+
+#### Milestone 3
```

Comment:
I suggest you mention the OSM tile server setup somewhere in this milestone as well. That is no small task.

Reply:
Fixed with [9344c53](https://github.com/ProximaEPFL/prd/pull/13/commits/9344c5342de58aee99d8e8bf64596d90e89a6fac)